winpr/libwinpr/synch/wait.c

```diff
@@ -61,7 +61,7 @@
 
 #ifndef _WIN32
 
-#include <alloca.h>
+#include <stdlib.h>
```

Comment:
Yes, linux include alloca.h in stdlib.h.

In man:
http://linux.die.net/man/3/alloca
#include <alloca.h>

https://www.freebsd.org/cgi/man.cgi?query=alloca&apropos=0&sektion=0&manpath=FreeBSD+10.0-RELEASE&arch=default&format=html
#include <stdlib.h>